### ISSUE TITLE
Error initializing classpath: No builders are available to build a model of type 'org.grails.gradle.plugin.model.GrailsClasspath'

### ISSUE THREAD

**Issue by chibante** [GMOD/Apollo#2020]
Thanks! I remove all java7 and the command `./apollo run-local` continue, but some steps after an error was found (at the end of this comment has all sequence of steps showed on terminal):
```
 | Error Error initializing classpath: No builders are available to build a model of type 'org.grails.gradle.plugin.model.GrailsClasspath'. (Use --stacktrace to see the full trace)
```
I check the status of grails on machine:
carla@einstein:~/Apollo$ `grails -version`
```
| Grails Version: 3.3.9
| JVM Version: 1.8.0_171
```
carla@einstein:~/Apollo$ `echo $GRAILS_HOME`
```
/home/carla/.sdkman/candidates/grails/current
```
**all sequence of steps**

carla@einstein:~/Apollo$ `./apollo run-local 8085`
```
No log file found in classpath.
From https://github.com/gmod/jbrowse
 * tag               1.15.4-release -> FETCH_HEAD
clean-webpack-plugin: /home/carla/Apollo/jbrowse/dist has been removed.
No log file found in classpath.
carla@einstein:~/Apollo$ cat saida.log 
Gradle version: 2
Using system gradle 2.11
Node Version: 11
Npm Version: 6
Yarn Version: 1
javac 1.8.0_171 found
javac installed
JDK 1.8 found: javac 1.8.0_171
Final JBrowse settings [git:[url:https://github.com/gmod/jbrowse, branch:1.15.4-release, alwaysPull:true, alwaysRecheck:true], plugins:[WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
Final plugins [WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]
:installJBrowseWebOnly
JBrowse already exists
Already up-to-date.
yarn install v1.13.0
[1/4] Resolving packages...
success Already up-to-date.
Done in 1.04s.
yarn install v1.13.0
info No lockfile found.
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
Done in 1.57s.
Installing JBrowse plugins  [WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]
Evaluating plugin WebApollo=[included:true]
Plugin jbrowse/plugins/WebApollo exists and appears valid.
Evaluating plugin RegexSequenceSearch=[included:true]
Plugin jbrowse/plugins/RegexSequenceSearch exists and appears valid.
Evaluating plugin HideTrackLabels=[included:true]
Plugin jbrowse/plugins/HideTrackLabels exists and appears valid.

> @gmod/jbrowse@1.15.4 build /home/carla/Apollo/jbrowse
> webpack

jbrowse-glob-loader: building 203 JBrowse modules
jbrowse-glob-loader: adding plugin module WebApollo/View/TrackList/Hierarchical
jbrowse-glob-loader: adding plugin module WebApollo/View/TrackList/Faceted
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/WebApolloCanvasVariants
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/WebApolloCanvasFeatures
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/WebApolloAlignments2
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/SequenceTrack
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/DraggableResultFeatures
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/DraggableHTMLVariants
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/DraggableHTMLFeatures
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/DraggableBLASTFeatures
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/DraggableAlignments
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/AnnotTrack
jbrowse-glob-loader: adding plugin module WebApollo/View/Track/AnnotSequenceTrack
jbrowse-glob-loader: adding plugin module WebApollo/View/Dialog/Help
jbrowse-glob-loader: adding plugin module WebApollo/TrackConfigTransformer
jbrowse-glob-loader: adding plugin module WebApollo/Store/SeqFeature/ScratchPad
jbrowse-glob-loader: adding plugin module WebApollo/Store/SeqFeature/PseudoNCList
jbrowse-glob-loader: adding plugin module WebApollo/Store/SeqFeature/ApolloGFF3
jbrowse-glob-loader: adding plugin module WebApollo/SequenceSearch
jbrowse-glob-loader: adding plugin module WebApollo/SequenceOntologyUtils
jbrowse-glob-loader: adding plugin module WebApollo/SequenceFeatureDialog
jbrowse-glob-loader: adding plugin module WebApollo/Permission
jbrowse-glob-loader: adding plugin module WebApollo/main
jbrowse-glob-loader: adding plugin module WebApollo/JSONUtils
jbrowse-glob-loader: adding plugin module WebApollo/InformationEditor
jbrowse-glob-loader: adding plugin module WebApollo/FormatUtils
jbrowse-glob-loader: adding plugin module WebApollo/FeatureSelectionManager
jbrowse-glob-loader: adding plugin module WebApollo/FeatureEdgeMatchManager
jbrowse-glob-loader: adding plugin module WebApollo/EUtils
jbrowse-glob-loader: adding plugin module WebApollo/BioFeatureUtils
jbrowse-glob-loader: adding plugin module RegexSequenceSearch/View/SearchSeqDialog
jbrowse-glob-loader: adding plugin module RegexSequenceSearch/Store/SeqFeature/RegexSearch
jbrowse-glob-loader: adding plugin module RegexSequenceSearch/main
jbrowse-glob-loader: adding plugin module NeatHTMLFeatures/View/Track/NeatFeatures
jbrowse-glob-loader: adding plugin module NeatHTMLFeatures/main
jbrowse-glob-loader: adding plugin module NeatCanvasFeatures/View/Track/NeatFeatures
jbrowse-glob-loader: adding plugin module NeatCanvasFeatures/View/FeatureGlyph/Segments
jbrowse-glob-loader: adding plugin module NeatCanvasFeatures/View/FeatureGlyph/ProcessedTranscript
jbrowse-glob-loader: adding plugin module NeatCanvasFeatures/View/FeatureGlyph/Gene
jbrowse-glob-loader: adding plugin module NeatCanvasFeatures/View/FeatureGlyph/Box
jbrowse-glob-loader: adding plugin module NeatCanvasFeatures/main
jbrowse-glob-loader: adding plugin module HideTrackLabels/main
jbrowse-glob-loader: adding plugin module DebugEvents/main
jbrowse-glob-loader: adding plugin module CategoryUrl/main
jbrowse-glob-loader: discovered a total of 247 modules
jbrowse-glob-loader: skipping entry point module JBrowse/main
Hash: cae210c880f002820eae
Version: webpack 3.12.0
Time: 27060ms
                    Asset       Size         Chunks                    Chunk Names
              0.bundle.js    8.16 MB  0, 1, 2, 3, 4  [emitted]  [big]  
              1.bundle.js     456 kB              1  [emitted]  [big]  
              2.bundle.js     145 kB              2  [emitted]         
              3.bundle.js     531 kB              3  [emitted]  [big]  
              4.bundle.js     398 kB              4  [emitted]  [big]  
              5.bundle.js    1.68 kB              5  [emitted]         
              6.bundle.js  920 bytes              6  [emitted]         
    run_jasmine.bundle.js     571 kB              7  [emitted]  [big]  run_jasmine
        browser.bundle.js     364 kB              8  [emitted]  [big]  browser
           main.bundle.js     375 kB              9  [emitted]  [big]  main
          0.bundle.js.map    11.3 MB  0, 1, 2, 3, 4  [emitted]         
          1.bundle.js.map     611 kB              1  [emitted]         
          2.bundle.js.map     200 kB              2  [emitted]         
          3.bundle.js.map     691 kB              3  [emitted]         
          4.bundle.js.map     504 kB              4  [emitted]         
          5.bundle.js.map    1.87 kB              5  [emitted]         
          6.bundle.js.map    1.13 kB              6  [emitted]         
run_jasmine.bundle.js.map     854 kB              7  [emitted]         run_jasmine
    browser.bundle.js.map     517 kB              8  [emitted]         browser
       main.bundle.js.map     537 kB              9  [emitted]         main
 dojo/resources/blank.gif   43 bytes                 [emitted]         
  [46] ./src/JBrowse/Browser.js 142 kB {0} [built]
  [65] ./src/JBrowse/Model/XHRBlob.js 7.2 kB {0} {3} [built]
 [129] ./build/dojo-webpack-plugin-loader/dojo/dojo.js 70.7 kB {7} {8} {9} [built]
 [334] ./src/JBrowse/Util.js 22.3 kB {0} {1} {3} [built]
 [336] ./src/JBrowse/QueryParamConfigMapper.js 6.13 kB {0} {2} [built]
 [337] ./src/JBrowse/standalone.js 140 bytes {0} {8} [built]
 [338] ./src/JBrowse/Model/SimpleFeature.js 2.86 kB {0} {1} [built]
 [340] ./src/JBrowse/View/GranularRectLayout.js 16.8 kB {0} {2} [built]
 [341] ./src/JBrowse/FeatureFiltererMixin.js 3.19 kB {0} [built]
 [342] ./src/JBrowse/View/Track/_AlignmentsMixin.js 15.8 kB {0} [built]
 [343] ./src/JBrowse/Store/SeqFeature/_MismatchesMixin.js 12.6 kB {0} [built]
 [344] ./src/JBrowse/Model/NestedFrequencyTable.js 3.41 kB {0} {1} [built]
 [345] ./src/JBrowse/Store/TabixIndexedFile.js 3.01 kB {0} [built]
 [373] ./src/JBrowse/main.js 13 kB {9} [built]
 [374] ./tests/js_tests/main.js 1.58 kB {7} [built]
    + 1485 hidden modules
:javac
:gwtc

BUILD SUCCESSFUL

Total time: 2 mins 1.738 secs

This build could be faster, please consider using the Gradle Daemon: https://doc| Resolving Dependencies. Please wait...
Downloading https://services.gradle.org/distributions/gradle-2.11-all.zip
.
.
.
.
.
.
.
.
.
.
Unzipping /home/carla/.gradle/wrapper/dists/gradle-2.11-all/ace7yl0a7udokebb27mt9s3qr/gradle-2.11-all.zip to /home/carla/.gradle/wrapper/dists/gradle-2.11-all/ace7yl0a7udokebb27mt9s3qr
Set executable permissions for: /home/carla/.gradle/wrapper/dists/gradle-2.11-all/ace7yl0a7udokebb27mt9s3qr/gradle-2.11/bin/gradle
Final JBrowse settings [git:[url:https://github.com/gmod/jbrowse, branch:1.15.4-release, alwaysPull:true, alwaysRecheck:true], plugins:[WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]]
Final plugins [WebApollo:[included:true], RegexSequenceSearch:[included:true], HideTrackLabels:[included:true]]

BUILD SUCCESSFUL

Total time: 14.51 secs
| Error Error initializing classpath: No builders are available to build a model of type 'org.grails.gradle.plugin.model.GrailsClasspath'. (Use --stacktrace to see the full trace)
```

_Originally posted by @chibante in https://github.com/GMOD/Apollo/issues/2016#issuecomment-456495873_

**Comment by nathandunn** [comment]
@chibante  Please add the notes directly to the github issue via the GitHub website.   I'm not entirely sure what the problem is now (or if there is one).  

Thanks

**Comment by nathandunn** [comment]
Can this be closed in favor of #2016

**Comment by nathandunn** [comment]
Closing this as Grails 2.5 should be used.
